src/Utils/GalleryUtils.ts

```diff
@@ -239,8 +239,8 @@ export function downloadItem(
         useDialog
           .getState()
           .showOkModalDialog(
-            "Failed to Connect",
-            "Failed to connect to temporary workspace. Please refresh the page and try again."
+            "Failed to connect",
+            "Failed to connect to temporary workspace. This could happen because of network issues. Please refresh the page and try again."
```

Comment:
do we need to mention this? I dont think we should. It could be cos of anything but we cant list all of them, please remove. seems odd

Reply:
Deborah suggested this text. Will remove for now